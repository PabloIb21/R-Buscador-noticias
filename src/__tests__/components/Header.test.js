import React from 'react';
import { mount } from 'enzyme';
import Header from '../../components/Header';

describe('<Header />', () => {
  test('mount', () => {
    const wrapper = mount(<Header titulo="Hola Mundo" />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('.brand-logo').text()).toBe('Hola Mundo');
  });
});