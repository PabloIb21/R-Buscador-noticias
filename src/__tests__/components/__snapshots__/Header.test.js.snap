// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> mount 1`] = `
<nav
  className="nav-wrapper light-blue darken-3"
>
  <a
    className="brand-logo center"
    href="#!"
  >
    Hola Mundo
  </a>
</nav>
`;
